ethereum-cpp-on-ubuntu/azuredeploy.parameters.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "storageAccountNamePrefix": {
+      "value": "GEN_UNIQUE"
```

Comment:
@ililic We found a bug in our CI with this. The unique string generated here was more than 11 chars allowed in the parameter which led to CI failures. So now the function is updated to let you specify how many characters to generate by following this format: GEN_UNIQUE_[N]. In this case, GEN_UNIQUE_11 since these will be concatenated with the 13 chars of uniqueString() function to make the allowed 24 chars for storage account names.